lib/features/mailbox/presentation/model/mailbox_categories.dart

```diff
@@ -17,6 +18,8 @@ extension MailboxCategoriessExtension on MailboxCategories {
         return 'exchange';
       case MailboxCategories.folders:
         return 'folders';
+      case MailboxCategories.appGrid:
+        return 'appGrid';
```

Comment:
using enum class

Reply:
It still is extension before enum class support, please make a move